docs/subquery_network/testnet/indexers/become-an-indexer.md

```diff
@@ -50,11 +50,11 @@ Make sure the indexer service versions are correct:
 Please change the default PostgreSQL password in the `POSTGRES_PASSWORD` field and in the coordinator-service's `postgres-password` field. Replace it with your own one. 
 :::
 
-Note:
-
+:::info Note
```

Comment:
- This note will be useful when the user will be visiting this page and particularly this section to check the version. As a user may have be redirected from either local installation, aws installation, or Linux installation guide, these inks herewith allow them to directly jump to the next section (in their respective guides). 

- However, we don't want those users who are just getting a brief of the process to directly jump to these given links. Hence, we can omit this highlight if needed. Let me know your suggestions on this.

Reply:
@ZeelPatel1102 good idea, I agree. I will reverse changes here.